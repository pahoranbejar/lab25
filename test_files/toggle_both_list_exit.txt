
What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 0

[Available] Red Scooter: Red scooter. Has battery.
[Available] Blue Scooter: Battery-powered scooter.

What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 1

Which item do you want to check in/out?
(0) Red Scooter
(1) Blue Scooter
Option: 0

What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 0

[Checked Out] Red Scooter: Red scooter. Has battery.
[Available] Blue Scooter: Battery-powered scooter.

What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 1

Which item do you want to check in/out?
(0) Red Scooter
(1) Blue Scooter
Option: 1

What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 0

[Checked Out] Red Scooter: Red scooter. Has battery.
[Checked Out] Blue Scooter: Battery-powered scooter.

What would you like to do?
(0) List items
(1) Check item in/out
(2) Quit
Option: 2
