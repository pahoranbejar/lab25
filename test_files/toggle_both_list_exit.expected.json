{"Red Scooter": {"description": "Red scooter. Has battery.", "checked out": true}, "Blue Scooter": {"description": "Battery-powered scooter.", "checked out": true}}